progs/symmetrc_mimc.py

```diff
@@ -0,0 +1,69 @@
+from random import randint
+from asyncio import gather
+from honeybadgermpc.field import GF
+from honeybadgermpc.mpc import PreProcessedElements, Subgroup
+from honeybadgermpc.elliptic_curve import Point, Jubjub
+from progs.jubjub import share_mul
+from progs.mimc import mimc_mpc, mimc_plain
+
+pp_elements = PreProcessedElements()
+field = GF(Subgroup.BLS12_381)
+CURVE = Jubjub()
+KEY_LENGTH = 32
```

Comment:
This should be a method parameter on encryption and decryption. I think having this be a default named argument is sufficient for now, but otherwise, see prof. miller's previous comment about this on your last PR